src/InputGroup/InputGroup.test.js

```diff
@@ -215,100 +139,54 @@ describe('<InputGroup />', () => {
 
     });
 
-    test('input text entered', () => {
-        const wrapper = mount(inputTextPosBefore).children().children();
-
-        // click up
-        wrapper.setState({ value: '' });
-        wrapper
-            .find('input[type="text"]')
-            .simulate('change', { target: { value: 'hello' } });
-
-        expect(wrapper.state('value')).toEqual('hello');
-    });
-
-    test('number input handle up and down', () => {
-        const wrapper = mount(numberInput).children().children();
-
-        // click up
-        wrapper.setState({ value: 0 });
-        wrapper
-            .find(
-                'button.fd-button--half.sap-icon--slim-arrow-up'
-            )
-            .simulate('click');
-        expect(wrapper.state('value')).toEqual(1);
-
-        wrapper
-            .find(
-                'button.fd-button--half.sap-icon--slim-arrow-down'
-            )
-            .simulate('click');
-        expect(wrapper.state('value')).toEqual(0);
-    });
-
-    describe('Prop spreading', () => {
-        test('should allow props to be spread to the InputGroup component (type text, addonPos after)', () => {
-            const element = mount(<InputGroup data-sample='Sample' />);
-
-            expect(
-                element.getDOMNode().attributes['data-sample'].value
-            ).toBe('Sample');
-        });
-
-        test('should allow props to be spread to the InputGroup component\'s (type text, addonPos after) input element', () => {
-            const element = mount(<InputGroup inputProps={{ 'data-sample': 'Sample' }} />);
-
-            expect(
-                element.find('input').getDOMNode().attributes['data-sample'].value
-            ).toBe('Sample');
-        });
-
-        test('should allow props to be spread to the InputGroup component for addonPos before (type text)', () => {
-            const element = mount(<InputGroup addonPos='before' data-sample='Sample' />);
+    let setup = (props) => {
+        return mount(<InputGroup {...props}>
+            <InputGroup.Addon>$</InputGroup.Addon>
+            <InputGroup.Input />
+        </InputGroup>);
+    };
 
-            expect(
-                element.getDOMNode().attributes['data-sample'].value
-            ).toBe('Sample');
+    describe('Default Rendering', () => {
+        let element = setup({
+            children: 'foo' //prop children is required
```

Comment:
This seems weird - shouldn't `InputGroup.Addon` and the `FormInput` be the children?